app/code/OleksiiBezpoiasnyi/RegularCustomer/view/frontend/layout/catalog_product_view.xml

```diff
@@ -8,7 +9,8 @@
             >
                 <arguments>
                     <argument name="sort_order" xsi:type="string">40</argument>
-                    <argument translate="true" name="title" xsi:type="string">Loyalty program for Regular Customers</argument>
+                    <argument translate="true" name="title" xsi:type="string">Loyalty program for Regular Customers
```

Comment:
@AlexeyBezpoyasny Never do like this in XML without a reason! The text
> Loyalty program for Regular Customers

is not the same as this:

> Loyalty program for Regular Customers
> \--------------------


Adding newline and spaces is another string.